boa/ipython.py

```diff
@@ -23,6 +23,7 @@ def deployer(self, line, cell):
         if line:
             self.shell.user_ns[line] = c  # ret available in user ipython locals
             _contracts[line] = c  # ret available in boa.ipython._contracts
+        global _
```

Comment:
is this necessary?

Reply:
yes, otherwise the tests fail: https://github.com/vyperlang/titanoboa/blob/0a8640f049f8b4703f09b998eaef36be01be0316/tests/unitary/test_ipython.py#L23

Reply:
i'm not really sure we need the `boa.ipython._` thing btw, since ipython already has the global `_` thing working. i don't really see anybody use it, so maybe we could remove it?

Reply:
https://github.com/vyperlang/titanoboa/pull/183/commits/0e9825bd4e41192e86d7343ea7bed9d3bff45c0e